dcm/src/main/java/com/vmware/dcm/backend/ortools/OrToolsSolver.java

```diff
@@ -193,9 +197,21 @@ public Builder setPrintDiagnostics(final boolean printDiagnostics) {
             return this;
         }
 
+        /**
+         * Configures whether or not to use presence literals in the capacity constraint function
```

Comment:
@lalithsuresh I did my best to write an informative comment, but my understanding of what presenceLiterals represent is not the best, so I wanted to ask if there's more information that should be included here?

Reply:
You may add:

```
If true, a capacity constraint is allowed to fail (that is, it is possible that some elements cannot be assigned to their domains in a way that satisfy capacity constraints).
```

Reply:
And do make the change as a separate commit so I can follow the diffs (rather than force pushing onto the same commit). When I merge, I'll squash all commits into one anyway.